1.8/administration/basic-guideline

```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title:  "Basic Guideline"
+title:  "First steps"
```

Comment:
Perhaps change the file name to match this as well?

Reply:
I tought I did that... Well, sorry, my bad...